catalog/pom.xml

```diff
@@ -551,15 +551,4 @@
         <module>confluence</module>
         <module>async</module>
     </modules>
-    <build>
-        <plugins>
-            <plugin>
-                <artifactId>maven-surefire-plugin</artifactId>
-            </plugin>
-            <plugin>
```

Comment:
❓ Confirming that we're still getting jacoco to run. I suspect this was orphaned here long ago and didn't need to remain, but it'd be good to know that nothing has changed in that regard.

Reply:
Jacoco is still running when that build option is taken out.

Proof: when running `mvn test` in `ddf/catalog`, I saw `[INFO] --- jacoco-maven-plugin:0.8.1:prepare-agent (default-prepare-agent) @ ...` appear multiple times.